contexts/credentials/examples/v1

```diff
@@ -42,6 +44,8 @@
     "evidenceDocument": "ex:evidenceDocument",
     "spouse": "schema:spouse",
     "subjectPresence": "ex:subjectPresence",
-    "verifier": {"@id": "ex:verifier", "@type": "@id"}
+    "verifier": {"@id": "ex:verifier", "@type": "@id"},
+    "currentStatus": "ex:currentStatus",
+    "statusReason": "ex:statusReason",
```

Comment:
No trailing commas in JSON.

```suggestion
    "statusReason": "ex:statusReason"
```

Reply:
This comma must be removed before merge!